libs/template/templates/default/library/versions.tmpl

```diff
@@ -11,16 +12,39 @@
    * only connect to compute of that same version and higher. If the version is
    * deemed too old, customers can update the version themselves after initializing
    * the template.
+   *
+   * See https://docs.databricks.com/dev-tools/databricks-connect/python/index.html#requirements
+   * for DB Connect release notes and version compatibility.
```

Comment:
Can it make sense to change how DB Connect version is selected for serverless-only bundles?